lib/core/Playlist.js

```diff
@@ -7,6 +8,55 @@ class Playlist extends Feed {
         this.#client = client;
     }
 
+    /**
+     * @returns {import('../parser/contents/PlaylistSidebarPrimaryInfo') | undefined}
+     */
+    getPrimaryInfo() {
+        return Simplify.matching({
+            type: Simplify.matching(/^PlaylistSidebarPrimaryInfo$/),
+        }).runOn(this.page)[0];
+    }
+
+    #getStat(index) {
+        const primaryInfo = this.getPrimaryInfo();
+        if (!primaryInfo || !primaryInfo.stats)
+            return 'N/A';
+        return primaryInfo.stats[index]?.toString() || 'N/A';
+    }
+
+    get title() {
+        const primaryInfo = this.getPrimaryInfo();
+        if (!primaryInfo)
+            return '';
+        return primaryInfo.title.toString();
+    }
+
+    get description() {
+        const primaryInfo = this.getPrimaryInfo();
+        if (!primaryInfo)
+            return '';
+        return primaryInfo.description.toString();
+    }
+
+    get total_items() {
+        return this.#getStat(0);
+    }
+
+    get views() {
+        return this.#getStat(1);
+    }
+
+    get last_updated() {
+        return this.#getStat(2);
+    }
+
+    /**
+     * @alias videos
+     */
+    get items () {
+        return this.videos;
```

Comment:
Still need to confirm that the new `SimpleVideo` interface is 100% compatible with the old `PlaylistItem` interface

Reply:
Also, pretty sure this parser does not yet work with ytmusic, will need to test